app/views/registers/index.html.haml

```diff
@@ -48,7 +48,7 @@
   - else
     .grid-row
       .column-full
-        %h1 Registers pipeline
+        %h1.heading-xlarge Registers pipeline
```

Comment:
did you intend this change to go in there?

Reply:
Yeah. #cleanasyougo